doc/04-Getting-started.md

```diff
@@ -28,6 +28,67 @@ this is an advanced task you should not tackle at the early beginning.
 
 Take some time to really understand how to work with Icinga Director first.
 
+First steps on the Frontend
+====================================
+Create a host
+------------------
+
+First we create a `host template` which we will name `director-generic-host`. The default check we will use for a Host is `hostalive`.
+
+Click on the interface on Icinga Director > Host Objects > Add Host and `add a template`. For details have a look at the screenshots below:
+
+![add host section](screenshot/director/04-getting-started/01-add_a_host.png)
+
+![add host template](screenshot/director/04-getting-started/02-add_host_template.png)
+
+Now we can add our first host. Go to Icinga Director > Host Objects > Add Host and choose `Object`:
+
+![add host object](screenshot/director/04-getting-started/03-add_a_host_object.png)
+
+Deploy your Config
+------------------
+
+It is time to deploy our first configuration. Click on 
+Deployments/ History > Render config > Deploy to master
+
+![deploy config](screenshot/director/04-getting-started/04-render_and_deploy_a_config.png)
+
+It can be that you get an error while deploying your config. This is highlighted by the red X. Read the `Startup log` carefully and if it is a duplicate declaration, comment out the mentioned lines in your already existing config. Do not restart or reload the master while doing that. This would result in a breakdown of the API and won't make it possible to redeploy again. After canging the config, try redeploying again. 
+
+![deployment details](screenshot/director/04-getting-started/05-deployment_details.png)
+
+Go on
+-----
+
+All your other objects can be created the same way. Just take over your existing objects step by step.
+
```

Comment:
While I'd love too see those documented the jump from a basic host to custom datafields is IMO too fast here. There should/will be a dedicated section named "Working with custom variables". You could consider working on that one if you want, but please contact me for a rough structure before you start doing so. In the meantime I'd suggest to remove this right now, as it would delay this pull request.

What's IMO completely missing is the "quick" way to custom fields when starting with command arguments - something you completely skipped. Vars/fields relations and their inheritance (command -> host -> service) also needs explanation. People should have deeper understanding of what they do before they blindly start creating fields.